README.md

```diff
@@ -27,6 +27,20 @@ driver: {
 }
 ```
 
+The argument can either be a string which specifies the location to save the screenShots like  'path:report' or 
+ an Object which has the following properties
+{
+        "screenShotPath" : "path:report",
+        "autoCaptureOptions" : ["click","exception"]
+}
+
+####The autocapture options
```

Comment:
Are there more than these two events where possibly a user would want to auto-capture? If so, I would recommend that the user be able to enter any of those possible event IDs here.